sql/core/src/test/scala/org/apache/spark/sql/test/SharedSparkSession.scala

```diff
@@ -72,6 +72,8 @@ trait SharedSparkSessionBase
       // this rule may potentially block testing of other optimization rules such as
       // ConstantPropagation etc.
       .set(SQLConf.OPTIMIZER_EXCLUDED_RULES.key, ConvertToLocalRelation.ruleName)
+      // Reduce shuffle partitions to reduce testing time.
+      .set(SQLConf.SHUFFLE_PARTITIONS.key, "5")
```

Comment:
I think I am hesitant since the affected scope is almost global. Let's see the test results first.

Reply:
```
[info] - recover from a Spark v2.1 checkpoint *** FAILED *** (1 second, 548 milliseconds)
[info]   Expected exception org.apache.spark.sql.streaming.StreamingQueryException to be thrown, but no exception was thrown (StreamSuite.scala:776)
[info]   org.scalatest.exceptions.TestFailedException:
```

Yes. This test failure related to [this commit](https://github.com/apache/spark/pull/25891/commits/3dc012475274d609e673ca5ceb71b9c3c501cb1f).